library/core/src/ptr/const_ptr.rs

```diff
@@ -3,6 +3,10 @@ use crate::cmp::Ordering::{Equal, Greater, Less};
 use crate::intrinsics::const_eval_select;
```

Comment:
Can we discard changes in this file since you are working on offset_from for only mut ptrs? This probably came up since you branched off ptr_const I think. Please branch off of main.